data/style.css

```diff
@@ -98,10 +98,6 @@ headerbar.flat.no-title .title {
 	background: @dialog_bg_color;
 }
 
-.large .header-image {
```

Comment:
This was mostly meant for the focus ring, but it looks like it broke at some point, I'll fix it in a separate PR, so I'll revert this